### ISSUE TITLE
UI: As a user / future participant I can click the "RSVP" button, I will receive an email with a confirmation, and I'll be added to the public list of event attendees.

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/chapter#278]
Depends on #110

**Comment by allella** [comment]
For simplicity, it may be best for the MVP to have a ["Sign in with Google" button both in the navigation, as suggested by Quincy](https://github.com/freeCodeCamp/chapter/issues/306#issuecomment-568205711) AND in place of the the RSVP form if the _user_ is not logged in.

Otherwise, if we allowed a non-authenticated _user_ to start an _rsvp_ then we'd have to store their input somewhere until after they successfully authenticate.  That may introduce unnecessary complexity for the MVP. 

If the _user_ clicks the "Sign In with Google" button, then _presumably_ Google can be handed a return URI to which the _user_ will be returned.

The user can then proceed to filling out any _rsvp_ fields for the _event_.

**Comment by QuincyLarson** [comment]
This is what Mail for Good's email key setup page looks like for reference:

<img width="804" alt="Mail_for_Good" src="https://user-images.githubusercontent.com/985197/73125755-091cf080-3f70-11ea-8a5f-a92dcc7856f3.png">

**Comment by QuincyLarson** [comment]
Closing in favor of #110
